src/docs/docs/guides/eslint-config-migration.md

```diff
@@ -82,6 +82,6 @@ The whole team should be committed to fixing the errors, don't make it to be a o
 
 # Think, have questions
 
-It's hard to have a motivation for this if you don't understand the purpose. If it's unclear for you why some rule was added into `@kiwicom/eslint-config` don't be afraid to ask on Slack in `#monorepo` channel.
+It's hard to have a motivation for this if you don't understand the purpose. If it's unclear for you why some rule was added into `@adeira/eslint-config` don't be afraid to ask on Slack in `#monorepo` channel.
```

Comment:
To ask in slack might not be valid anymore, create a github issue insteand maybe?

Reply:
Docs need to be updated a lot - you can skip it.